tests/on_target/tests/test_conn_bridge.py

```diff
@@ -7,21 +7,37 @@
 import time
 import os
 import types
+import glob
 from utils.flash_tools import flash_device, reset_device, recover_device, setup_jlink, dfu_device
 from utils.uart import Uart
 from tests.conftest import get_uarts
 import sys
 sys.path.append(os.getcwd())
 from utils.logger import get_logger
 
+logger = get_logger()
+
+def get_first_match_artifact(pattern):
```

Comment:
Can be moved to utils since it's used by more than one test.

Reply:
but where would you locate it? we dont really have a file for it

Reply:
flash_tools.py maybe? it's related to flashing I guess.

Reply:
yeah maybe that is close enough. will do